### PR TITLE
DE-48156 Improve ES requests logging

### PR DESCRIPTION
**Pull request by dorrogeray**
improved-logging

### CODE REVIEW

**Review by vojtahavranek — APPROVED** [review]
👍

File: src/Client.php [hunk 1]
```diff
@@ -25,6 +25,14 @@
  */
 class Client
 {
+    public const LOG_DISABLED = 0b0000;
```
Comment (davidpilny) [inline src/Client.php:28]: These are some special params of the ES?

File: src/Client.php [hunk 2]
```diff
@@ -554,11 +592,27 @@ public function request(string $path, string $method = Request::GET, $data = [],
             return $this->request($path, $method, $data, $query);
         }
 
-        $this->_logger->debug('Elastica Request', [
-            'request' => $request->toArray(),
-            'response' => $response->getData(),
-            'responseStatus' => $response->getStatus(),
-        ]);
+        if ($this->shouldLog()) {
+            $elapsedTimeMs = (int)(round($response->getQueryTime() * 1000));
```
Comment (davidpilny) [inline src/Client.php:596]: Maybe extract to separate method logRequest?

**Review by davidpilny — APPROVED** [review]

File: src/Client.php (hunk 1 above)
Comment (dorrogeray) [reply src/Client.php:28]: No, this is just a way to write readable flags. `0b0000` is unsigned int 0, `0b0001` is unsigned int 1, `0b0010` is unsigned int 2, `0b0011` is unsingned int 3, etc.

File: src/Client.php (hunk 2 above)
Comment (dorrogeray) [reply src/Client.php:596]: I will if this method gets beefier. :+1:

**Pull request merged**